blocks/event-format-component/event-format-component.js

```diff
@@ -26,21 +29,19 @@ async function decorateCloudTagSelect(column) {
   // const clouds = await getClouds();
 
   // if (!clouds) return;
+  // FIXME: use correct data source rather than hardcoded values.
+  const clouds = ['Creative Cloud', 'DX'];
```

Comment:
Let's take stage's code for this function. What we get from Chimera isn't working for our backend service.